mpd/mpd.go

```diff
@@ -74,12 +75,14 @@ type AdaptationSet struct {
 
 // Constants for DRM / ContentProtection
```

Comment:
Could we add a comment here with the url where these come from? http://dashif.org/identifiers/protection/ I think